game/singleton.hh

```diff
@@ -4,7 +4,7 @@
 template <class T> class Singleton {
   protected:
 	/// pointer to object
-	static T* ms_Singleton;
+	inline static T* ms_Singleton{};
```

Comment:
useless

Reply:
I added some comments in the code: this singleton is completely broken, and as singleton is an antipattern, better remove it than doing half baked fixes